scalability_and_performance/cluster-compare/troubleshooting-cluster-comparisons.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="troubleshooting-cluster-comparisons"]
+= Troubleshooting cluster comparisons
+include::_attributes/common-attributes.adoc[]
+:context: troubleshooting-cluster-comparisons
+
+toc::[]
+
+When using the `cluster-compare` plugin, you might see unexpected results, such as false positives or conflicts when multiple cluster custom resources (CRs) exist. 
+
+[id="troubleshooting-cc-false-positives_{context}"]
+== Troubleshooting false positives for missing resources
+
+The plugin might report a missing resource even though the cluster CR is present in the cluster.
+
+.Procedure
+
+. Ensure you are using the latest version of the `cluster-compare` plugin. See the section _Installing the cluster-compare plugin_.
+
+. Ensure you are using the most up-to-date version of the reference configuration.
+
+. Ensure that template has the same `apiVersion`, `kind`, `name`, and `namespace` fields as the cluster CR.
+
+[id="troubleshooting-cc-multiple-matches_{context}"]
+== Troubleshooting multiple template matches for the same CR
+
+In some cases, more than one cluster CR can match a template because they feature the same `apiVersion`, `namespace`, and `kind`. The plugin's default matching compares the CR that features the least differences.
+
+You can optionally configure your reference configuration to avoid this situation.
+
+.Procedure
+
+. Ensure the templates feature distinct `apiVersion`, `namespace`, and `kind` values to ensure no duplicate template matching.
+
+. Use a user configuration file to manually match a template to a CR. For more information, see _Performing advanced reference configuration customization_.
```

Comment:
Would it be possible to provide a link to this section?

Reply:
Added links in an additional resources section